cl/phase1/stages/stage_history_download.go

```diff
@@ -144,8 +144,8 @@ func SpawnStageHistoryDownload(cfg StageHistoryReconstructionCfg, ctx context.Co
 			}
 		}
 		isInElSnapshots := true
-		frozenBlocksInEL := cfg.engine.FrozenBlocks(ctx)
 		if blk.Version() >= clparams.BellatrixVersion && cfg.engine != nil && cfg.engine.SupportInsertion() {
+			frozenBlocksInEL := cfg.engine.FrozenBlocks(ctx)
```

Comment:
curious what's difference between these two? got panic before you move the line inside if-statement?